macosx/Makefile

```diff
@@ -30,16 +31,17 @@ NMAP_UPDATE_BUILD_DIR = nmap-update-build
 NMAP_UPDATE_STAGING_DIR = nmap-update
 
 JHBUILD_PREFIX=$(HOME)/gtk/inst
+OPENSSL_PREFIX=$(HOME)/gtk/source/openssl-1.0.2h
+export JHBUILD_PREFIX
 # MACPORTS_PREFIX is still here because of nmap-update, which needs
 # libsvn and libapr, which jhbuild doesn't support yet.
 MACPORTS_PREFIX=$(HOME)/macports-10.8
 PREFIX = /usr/local
 
-PACKAGEMAKER = /Developer/usr/bin/packagemaker
-
 EXTRA_DIST = README ../COPYING ../docs/3rd-party-licenses.txt ../docs/licenses
 
 CONFIGURE_ARGS = --prefix="$(PREFIX)" --with-libdnet=included --with-libpcap=included --with-libpcre=included --with-liblua=included CC="$(CC)" CXX="$(CXX)" CFLAGS="$(CFLAGS)" CXXFLAGS="$(CXXFLAGS)" LDFLAGS="$(LDFLAGS)" PYTHON="$(PYTHON)"
+NMAP_UPDATE_CONFIGURE_ARGS = --prefix="$(PREFIX)" --with-libdnet=included --with-libpcap=included --with-libpcre=included --with-liblua=included CC="$(CC)" CXX="$(CXX)" CFLAGS="$(CFLAGS) -I$(JHBUILD_PREFIX)/include/subversion-1" CXXFLAGS="$(CXXFLAGS)" LDFLAGS="$(LDFLAGS) -L$(JHBUILD_PREFIX)/lib -lsvn_client-1" PYTHON="$(PYTHON)"
```

Comment:
We shouldn't need to pass additional CXXFLAGS or other things to nmap-update's configure. The `--with-subversion` and `--with-apr` options should take care of that.